packages/amplify-codegen/src/commands/configure.js

```diff
@@ -3,6 +3,11 @@ const configureProjectWalkThrough = require('../walkthrough/configure');
 const add = require('./add');
 
 async function configure(context) {
+  try {
+    context.amplify.getProjectMeta();
+  } catch (e) {
+    context.withoutInit = true;
```

Comment:
Could we move this to a separate helper function instead. I am not a fan of adding more things to context

Reply:
Moving this as well as context.frontend and context.framework to parameters in the functions called from here